packages/web/public/images/badges/vote-recipient.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I see we already have badges in another folder. can we deduplicate?

Reply:
They are not the same, this badges are without borders.